trademapper/js/main.js

```diff
@@ -16,7 +16,8 @@ require.config({
 		// map data - jsons imported by require
 		countrycentre: "map/countrycentre",
 		disputedareas: "map/disputedareas",
-		worldmap: "map/worldmap"
+		worldmap: "map/worldmap",
+    caudata: "map/CAUDATA"
```

Comment:
Indentation.... Can your text editor to automatic `s/    /\t/g` on save?

Reply:
I've got it set to tabs permanently now. I probably could do that, but the issue then would be that some of the code mixes spaces and tabs, and it would introduce additional irrelevant whitespace changes.

My inclination is to go through all the files, replace tabs with spaces, then lint everything, but time...